### ISSUE TITLE
Increasing the batch size from default values

### ISSUE THREAD

**Issue by kristellmarisse** [weiliu89/caffe#154]
I noticed degraded performance in the trained model, when I reduced the batch size from default value and good performance when I trained with default settings. So for example, I increase the batch from 32 to 64, would it increase the performance (i,e detection accuracy)?

**Comment by weiliu89** [comment]
If you change batch size (I mean accum_batch_size), it might be worthwhile to explore slightly different initial learning rate. If you train with a base network that uses batch normalization (e.g. ResNet), using smaller batch size can affect the performance since the mean/variance might be not stable.
